build.gradle.kts

```diff
@@ -1,16 +1,13 @@
-import net.researchgate.release.GitAdapter.GitConfig
-import net.researchgate.release.ReleaseExtension
-
 description = "An application to reset the state of Kafka Connect connectors"
 
 plugins {
     `java-library`
-    id("net.researchgate.release") version "2.8.1"
+    id("net.researchgate.release") version "3.0.2"
     id("com.bakdata.sonar") version "1.1.11"
     id("com.bakdata.sonatype") version "1.1.11"
-    id("org.hildan.github.changelog") version "0.8.0"
-    id("com.google.cloud.tools.jib") version "3.3.0"
-    id("io.freefair.lombok") version "5.3.3.3"
+    id("org.hildan.github.changelog") version "1.12.1"
+    id("com.google.cloud.tools.jib") version "3.4.0"
+    id("io.freefair.lombok") version "6.6.3"
```

Comment:
Why not the `8.4`? If we upgrade to Java 21, version 6 will not be compatible.

Reply:
One step at a time. We need to check if our plugins work with Gradle 8. I also still need to test this PR, should have marked as draft :)

Reply:
I would test Gradle updates in another PR. Fine for you @raminqaf ?

Reply:
Tested the PR btw and it works